pkg/util/kubernetes/autoscalers/processor.go

```diff
@@ -32,6 +33,8 @@ const (
 	maxCharactersPerChunk = 7000
 	// extraQueryCharacters accounts for the extra characters added to form a query to Datadog's API (e.g.: `avg:`, `.rollup(X)` ...)
 	extraQueryCharacters = 16
+	// maxParallelQueries returns the maximum number of parallel queries to Datadog
+	maxParallelQueries = 300
```

Comment:
Curious to know on how we arrived to this number?

Reply:
It comes from internal testing on our largest clusters and just provide a (too high) ceiling just in case the features cause issues (which is not the case in our testing)

Reply:
Would be nice to add a comment 😄